commit 595814295b75e9cb5b0e6d774060e0910d6e376e (refs/changes/92/2892/1)
Author: Jared Morrow <jm@joyent.com>
Date:   2017-10-31T16:18:45-06:00 (1 year, 11 months ago)
    
    MANTA-3484 Binder's ZK timeout setting should be sessionTimeout with node-zkstream

diff --git a/lib/zk.js b/lib/zk.js
index f1fd475..5982c3b 100644
--- a/lib/zk.js
+++ b/lib/zk.js
@@ -26,7 +26,7 @@ function ZKCache(options) {
                 address: process.env.ZK_HOST || '127.0.0.1',
                 port: 2181,
                 log: options.log,
-                timeout: 30000
+                sessionTimeout: 30000
         });
         this.ca_domain = options.domain;
         this.ca_log = options.log;
